setup.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+"""
+Setup file for openedx-rocketchat Django plugin.
+"""
+from distutils.core import setup
+__version__ = '0.1.0'
+
+setup(
+    name='openedx-rocketchat',
+    version=__version__,
+    description='RocketChat extension for openedx',
+    author='eduNEXT',
```

Comment:
I don't know if the author needs to be Proversity instead. @andrey-canon

Reply:
I don't know either, but I will keep this, at least someone of proversity ask it